src/Plugin/Commerce/PaymentGateway/HostedPaymentFields.php

```diff
@@ -68,7 +69,8 @@ public function __construct(
     PaymentMethodTypeManager $payment_method_type_manager,
     TimeInterface $time,
     RounderInterface $rounder,
-    ClientFactory $client_factory
+    ClientFactory $client_factory,
+    FraudSession $fraud_session
```

Comment:
**CHANGE REQUEST**

When we have an interface we always use the interface in the declaration; we could be injecting a custom implementation of the service.

Like you have it on the payment method add form.

Reply:
used interface